src/components/3-Modeling/modelingText_mobile.vue

```diff
@@ -33,22 +33,22 @@
                     'bees_id': 'step_error_exp'
                 },
                 {
-                    'method': "In the hidden layer, the receiving nodes calculate a weighted sum of the received information…",
+                    'method': "In the hidden layer, the nodes calculate a weighted sum of the received information…",
                     'flubber_id': 'ANN4',
                     'bees_id': 'step_error_exp'
                 },
                 {
-                    'method': "…and then the weighted sum is adjusted by some simple rule, for example, keeping positive values as-is and setting negative values to zero.",
+                    'method': "…and then the weighted sum is adjusted by some simple rule, for example, dividing in half any weighted sums that are less than five.",
                     'flubber_id': 'ANN5',
                     'bees_id': 'step_error_exp'
                 },
                 {
-                    'method': "These output values are then passed to nodes in the next hidden layer, or, if there are no more hidden layers, to the output layer.",
+                    'method': "These adjusted values are then passed to nodes in the next hidden layer, or, if there aren't more hidden layers, to the output layer.",
                     'flubber_id': 'ANN6',
                     'bees_id': 'step_error_exp'
                 },
                 {
-                    'method': "Adding more nodes or hidden layers to the model increases the complexity of relationships between inputs and outputs that the model is able to mimic. To predict stream temperature, we use an ANN with many hidden layers.",
+                    'method': "The more nodes or hidden layers in the model, the better the model can “learn” complex relationships. To predict stream temperature, we use an ANN with many hidden layers.",
```

Comment:
is there a penalty for overfitting?

Reply:
Hmmm, I don't know whether there is a built-in penalty or if that's assessed by the modeler. ML models can certainly be overfit. I think we won't get into that complexity here. I see how my edit to the language maybe brings up that possibility more, though. We'll see what Sam thinks.